src/pages/Recipe/components/RecipeDescription/RecipeDescription.tsx

```diff
@@ -6,9 +7,11 @@ type RecipeDescriptionProps = {
   className?: string;
 };
 export const RecipeDescription: React.FC<RecipeDescriptionProps> = ({ description, className }) => {
+  /* eslint-disable import/namespace*/
+  const cleanDescription = DOMPurify.sanitize(description);
   return (
     <section className={className}>
-      <div className={styles.description} dangerouslySetInnerHTML={{ __html: description ?? '' }} />
+      <div className={styles.description} dangerouslySetInnerHTML={{ __html: cleanDescription ?? '' }} />
```

Comment:
Когда поправишь импорт у тебя тут подцепится тип string и не нужно будет дефолтное значение
```suggestion
      <div className={styles.description} dangerouslySetInnerHTML={{ __html: cleanDescription }} />
```